<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_pesquisaVenda</class>
 <widget class="QDialog" name="window_pesquisaVenda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>120</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Filtro</string>
   </property>
   <widget class="QRadioButton" name="rb_PeCodigo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Código</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_PeDescricao">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Descrição</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_PeBuscar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pesquisa</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_PePesquisa">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_PeProdutos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>431</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_PeSelecionarItem">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecionar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
